tests/test_adminpage.py

```diff
@@ -1,40 +1,45 @@
+from cms.api import create_page
 from cms.models import Page
+from cms.test_utils.testcases import CMSTestCase
 from django.contrib import admin
 
-from . import BaseTest
-
 page_admin = admin.site._registry[Page]
 
 
-class AdminPageTest(BaseTest):
+class AdminPageTest(CMSTestCase):
     def test_get_form_no_obj(self):
         """
         Test that the returned form has not been modified by the meta patch
         when no page object is specified
         """
-        request = self.get_page_request(None, self.user, "/", edit=True)
+        request = self.get_page_request(None, self.get_superuser(), "/")
         form = page_admin.get_form(request)
         self.assertEqual(form.base_fields.get("meta_description"), None)
 
     def test_get_form_with_obj(self):
         """
         Test that the returned form has been modified by the meta patch
         """
-        page1, _page2 = self.get_pages()
+        language = "en"
+        superuser = self.get_superuser()
+        page1 = create_page(title="test", template="page_meta.html", language=language)
 
-        request = self.get_page_request(page1, self.user, "/", edit=True)
+        request = self.get_page_request(page1, superuser, "/")
```

Comment:
user should be ok here, we should aim to change as little as possible to make bringing in updates easier in the future:

```suggestion
        request = self.get_page_request(page1, self.user, "/")
```